pkg/3scale/amp/operator/redis_reconciler.go

```diff
@@ -124,3 +147,100 @@ func Redis(apimanager *appsv1alpha1.APIManager, client client.Client) (*componen
 	}
 	return component.NewRedis(opts), nil
 }
+
+func (r *RedisReconciler) deleteSystemRedisSecretNamespaceAttribute() (reconcile.Result, error) {
+	redis, err := Redis(r.apiManager, r.Client())
+	if err != nil {
+		return reconcile.Result{}, err
+	}
+
+	desired := redis.SystemRedisSecret()
+	existing := &corev1.Secret{}
+	err = r.Client().Get(context.TODO(), types.NamespacedName{Name: desired.Name, Namespace: r.apiManager.Namespace}, existing)
+	if k8serr.IsNotFound(err) {
+		return reconcile.Result{}, nil
+	}
+	if err != nil {
+		return reconcile.Result{}, err
+	}
+
+	update := false
+	namespaceAttr := "NAMESPACE"
```

Comment:
small thing but this is a key, not an attribute - attributes are characteristics or properties of an object

Reply:
sure, sorry, yes, renamed all related things. Thank you